manim/utils/color/manim_colors.py

```diff
@@ -0,0 +1,91 @@
+LIGHT_GREY: ManimColor = ManimColor("#BBBBBB")
+GRAY: ManimColor = ManimColor("#888888")
+GREY: ManimColor = ManimColor("#888888")
+DARK_GRAY: ManimColor = ManimColor("#444444")
+DARK_GREY: ManimColor = ManimColor("#444444")
+DARKER_GRAY: ManimColor = ManimColor("#222222")
+DARKER_GREY: ManimColor = ManimColor("#222222")
+BLUE_A: ManimColor = ManimColor("#C7E9F1")
+BLUE_B: ManimColor = ManimColor("#9CDCEB")
+BLUE_C: ManimColor = ManimColor("#58C4DD")
+BLUE_D: ManimColor = ManimColor("#29ABCA")
+BLUE_E: ManimColor = ManimColor("#236B8E")
+PURE_BLUE: ManimColor = ManimColor("#0000FF")
+BLUE: ManimColor = ManimColor("#58C4DD")
+DARK_BLUE: ManimColor = ManimColor("#236B8E")
+TEAL_A: ManimColor = ManimColor("#ACEAD7")
+TEAL_B: ManimColor = ManimColor("#76DDC0")
+TEAL_C: ManimColor = ManimColor("#5CD0B3")
+TEAL_D: ManimColor = ManimColor("#55C1A7")
+TEAL_E: ManimColor = ManimColor("#49A88F")
+TEAL: ManimColor = ManimColor("#5CD0B3")
+GREEN_A: ManimColor = ManimColor("#C9E2AE")
+GREEN_B: ManimColor = ManimColor("#A6CF8C")
+GREEN_C: ManimColor = ManimColor("#83C167")
+GREEN_D: ManimColor = ManimColor("#77B05D")
+GREEN_E: ManimColor = ManimColor("#699C52")
+PURE_GREEN: ManimColor = ManimColor("#00FF00")
+GREEN: ManimColor = ManimColor("#83C167")
+YELLOW_A: ManimColor = ManimColor("#FFF1B6")
+YELLOW_B: ManimColor = ManimColor("#FFEA94")
+YELLOW_C: ManimColor = ManimColor("#FFFF00")
+YELLOW_D: ManimColor = ManimColor("#F4D345")
+YELLOW_E: ManimColor = ManimColor("#E8C11C")
+YELLOW: ManimColor = ManimColor("#FFFF00")
+GOLD_A: ManimColor = ManimColor("#F7C797")
+GOLD_B: ManimColor = ManimColor("#F9B775")
+GOLD_C: ManimColor = ManimColor("#F0AC5F")
+GOLD_D: ManimColor = ManimColor("#E1A158")
+GOLD_E: ManimColor = ManimColor("#C78D46")
+GOLD: ManimColor = ManimColor("#F0AC5F")
+RED_A: ManimColor = ManimColor("#F7A1A3")
+RED_B: ManimColor = ManimColor("#FF8080")
+RED_C: ManimColor = ManimColor("#FC6255")
+RED_D: ManimColor = ManimColor("#E65A4C")
+RED_E: ManimColor = ManimColor("#CF5044")
+PURE_RED: ManimColor = ManimColor("#FF0000")
+RED: ManimColor = ManimColor("#FC6255")
+MAROON_A: ManimColor = ManimColor("#ECABC1")
+MAROON_B: ManimColor = ManimColor("#EC92AB")
+MAROON_C: ManimColor = ManimColor("#C55F73")
+MAROON_D: ManimColor = ManimColor("#A24D61")
+MAROON_E: ManimColor = ManimColor("#94424F")
+MAROON: ManimColor = ManimColor("#C55F73")
+PURPLE_A: ManimColor = ManimColor("#CAA3E8")
+PURPLE_B: ManimColor = ManimColor("#B189C6")
+PURPLE_C: ManimColor = ManimColor("#9A72AC")
+PURPLE_D: ManimColor = ManimColor("#715582")
+PURPLE_E: ManimColor = ManimColor("#644172")
+PURPLE: ManimColor = ManimColor("#9A72AC")
+PINK: ManimColor = ManimColor("#D147BD")
+LIGHT_PINK: ManimColor = ManimColor("#DC75CD")
+ORANGE: ManimColor = ManimColor("#FF862F")
+LIGHT_BROWN: ManimColor = ManimColor("#CD853F")
+DARK_BROWN: ManimColor = ManimColor("#8B4513")
+GRAY_BROWN: ManimColor = ManimColor("#736357")
+GREY_BROWN: ManimColor = ManimColor("#736357")
```

Comment:
Is it necessary to define explicit types here? Afaik mypy will infer them correctly

Reply:
Having them untyped let's mypy ignore the type, but we need ignore-untyped-defs such that mypy doesn't die because of too many errors

Reply:
Ok I see, so this could be removed in the future then?